app/src/main/res/layout/dialog_filter.xml

```diff
@@ -15,4 +15,16 @@
         android:hint="@string/filter_add_description"
         app:layout_constraintTop_toTopOf="parent"
         />
+    <CheckBox
+        android:id="@+id/phraseWholeWord"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:paddingEnd="24dp"
+        android:paddingStart="24dp"
+        android:text="@string/filter_dialog_whole_word"
+        android:button="@null"
+        android:drawableRight="?android:attr/listChoiceIndicatorMultiple"
```

Comment:
This `android:button="@null"` and `android:drawableRight="?android:attr/listChoiceIndicatorMultiple"` messes up touch highlighting